Slim/Control/Queries.pm

```diff
@@ -5010,6 +5011,22 @@ sub _songData {
 			$remoteMeta->{T} = $remoteMeta->{samplerate};
 			$remoteMeta->{I} = $remoteMeta->{samplesize};
 			$remoteMeta->{W} = $remoteMeta->{releasetype};
+
+			$remoteMeta->{V} = -1;
+            # Distance from the live edge of live remote stream. -1 is not live, 0 is live at the edge, >0 is distance in seconds from the live edge.
+			if ( my $client = $request->client ) {
+				if (my $song = $client->currentSongForUrl($url)) {
```

Comment:
Both `$client` and `$song` are used in other places of this sub. Please put the definition of the two at the top and remove this repeated evaluation of the two values from wherever it's done now (within this sub, eg. line 5017).